docker/dockerfiles/entrypoint.sh

```diff
@@ -1,19 +1,14 @@
 #!/bin/sh
 
-cd /home/circleci/code-dot-org
+cd $HOME/code-dot-org
 
 # Run https://github.com/boxboat/fixuid allow writes to bind-mounted code-dot-org directory
 eval $( fixuid )
 
 # Need to change ownership of volume mounts which are not bind-mounted to the uid/gid after fixuid is applied
-sudo chown -R circleci:circleci /home/circleci/.rbenv \
-    /home/circleci/.config \
-    /home/circleci/.cache
+sudo chown -R $USER:$GROUP \
+        $HOME/.rbenv \
+        $HOME/.config \
+        $HOME/.cache
 
 eval "$(rbenv init -)"
-
-# start mysql
-sudo service mysql start && mysql -V
-
-# execute original command
-exec "$@"
```

Comment:
How does [invoking tests like `/entrypoint.sh docker/unit_tests.sh || EXIT_CODE=$?`]( https://github.com/code-dot-org/code-dot-org/pull/51630/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R70) work without this `exec` line?

Reply:
gah, I mismerged somewhere here, TY for catching 🙏🙏🙏